spec/lib/annotate/annotate_models_spec.rb

```diff
@@ -915,13 +915,13 @@ def mock_column(name, type, options = {})
     end
   end
 
-  describe '#get_patterns' do
+  describe '.get_patterns' do
     subject { AnnotateModels.get_patterns(options, pattern_type) }
 
-    context 'when pattern_type=additional_file_patterns' do
+    context 'when pattern_type is "additional_file_patterns"' do
       let(:pattern_type) { 'additional_file_patterns' }
 
-      context 'with additional_file_patterns' do
+      context 'when additional_file_patterns is specified is the options' do
```

Comment:
Same as before, did you mean to put `'when additional_file_patterns is specified in the options'` is => in

Reply:
Yes, I mean to put `is specified in the options`...

Reply:
Same as before, I mean to put is specified in the options...